apps/entity-service/src/entities/project-pitch.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable, NotFoundException } from "@nestjs/common";
+import { ProjectPitch, User } from "@terramatch-microservices/database/entities";
+import { Includeable, Op } from "sequelize";
+import { PaginatedQueryBuilder } from "@terramatch-microservices/database/util/paginated-query.builder";
+import { MAX_PAGE_SIZE } from "./entities.service";
+import { EntityQueryDto } from "./dto/entity-query.dto";
+
+@Injectable()
+export class ProjectPitchService {
+  async getProjectPitch(uuid: string) {
+    const projectPitch = await ProjectPitch.findOne({ where: { uuid } });
+    if (!projectPitch) {
+      throw new NotFoundException("ProjectPitch not found");
+    }
+    return projectPitch;
+  }
+
+  async getProjectPitches(userId: number, params: EntityQueryDto) {
+    const user = await User.findOne({
+      include: ["organisations"],
```

Comment:
Organisations is a bit table; please limit this to the attributes you need. Looks like it's just UUID